### PR TITLE
{math}[foss/2020b,intel/2020b] SUNDIALS v6.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3594.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e2ff3ca3fc876a3491d6a9aa5a2cbafa for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/00f8bc836f2e973606a8866b78e6a419 for a full test report.

File: easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-6.2.0-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'intel', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'usempi': True, 'pic': True}
+
+source_urls = ['https://github.com/LLNL/sundials/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['195d5593772fc483f63f08794d79e4bab30c2ec58e6ce4b0fb6bcc0e0c48f31d']
+
+# two valid checksums: one for original source tarball downloaded from computation.llnl.gov,
+# other for new source tarball downloaded from GitHub
+
```
Comment (branfosj) [inline easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-6.2.0-intel-2020b.eb:19]:
```suggestion
```

File: easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-6.2.0-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'usempi': True, 'pic': True}
+
+source_urls = ['https://github.com/LLNL/sundials/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['195d5593772fc483f63f08794d79e4bab30c2ec58e6ce4b0fb6bcc0e0c48f31d']
+
+# two valid checksums: one for original source tarball downloaded from computation.llnl.gov,
+# other for new source tarball downloaded from GitHub
+
```
Comment (branfosj) [inline easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-6.2.0-foss-2020b.eb:19]:
```suggestion
```

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/4305606e10e790c8070fe050d5a5224d for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**